xml/ns-System.Formats.Asn1.xml

```diff
@@ -1,6 +1,11 @@
 <Namespace Name="System.Formats.Asn1">
   <Docs>
     <summary>Contains types used in reading and writing Abstract Syntax Notation One (ASN.1) data structures.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+## Remarks
+For more information, see [architecture document](https://github.com/dotnet/designs/blob/main/accepted/2020/asnreader/asnreader.md).
```

Comment:
```suggestion

For more information, see the [ASN.1 BER/CER/DER Reader & Writer](https://github.com/dotnet/designs/blob/main/accepted/2020/asnreader/asnreader.md) design document.
```

Reply:
since `document` is the simple noun here, wouldn't all of `ASN.1 BER/CER/DER Reader and Writer design document` be the link text (as the full noun phrase)?

Reply:
@bartonjs Yes that's better. I updated my suggestion.

Reply:
I updated the PR